projects/admin/src/app/routes/items-route.ts

```diff
@@ -54,13 +54,28 @@ export class ItemsRoute extends BaseRoute implements RouteInterface {
             canRead: (record: any) => this.canReadItem(record),
             canUpdate: (record: any) => this._routeToolService.canUpdate(record, this.recordType),
             canDelete: (record: any) => this._routeToolService.canDelete(record, this.recordType),
+            preprocessRecordEditor: (record: any) => {
+              const holdingPid = this._routeToolService.getRouteQueryParam('holding');
```

Comment:
Comments are welcome here to summary the function.

Reply:
done